test/commands/test/resources/broken/nunit_report.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be good for me to understand what causes this file to be broken - have we manually broken it? Or is this the example .xml as it was provided from the failing CLI run and we previously just weren't flagging up the issue?

Reply:
It's a manually broken file

Reply:
Part of me was hoping it was something from the intermittent issue we've been seeing 🙂